tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/StackTraceLeak/DeveloperExceptionPageMiddlewareIntegration.cs

```diff
@@ -0,0 +1,49 @@
+// <copyright file="DeveloperExceptionPageMiddlewareIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+#if !NETFRAMEWORK
+
+using System;
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+using Microsoft.AspNetCore.Http;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.StackTraceLeak;
+
+/// <summary>
+/// DeveloperExceptionPageMiddleware integration
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Microsoft.AspNetCore.Diagnostics",
+    TypeName = "Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware",
+    ParameterTypeNames = new[] { "Microsoft.AspNetCore.Http.HttpContext", ClrNames.Exception },
+    MethodName = "DisplayException",
+    ReturnTypeName = ClrNames.Task,
+    MinimumVersion = "2.0.0.0",
+    MaximumVersion = "2.*.*.*",
```

Comment:
I wasn't sure because I saw both 3 or 4 points in other instrumentations. It looks like the 3 points option is more popular, so I will go for that. Thanks!

Reply:
Yeah, it really doesn't matter I think, just wondered 😃